recipes/mathter/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,16 @@
+//L=============================================================================
+//L This software is distributed under the MIT license.
+//L Copyright 2021 P�ter Kardos
```

Comment:
```suggestion
//L Copyright 2021 Peter Kardos
```
There seem to be an encoding issue

Reply:
This copyright should be removed anyway.

Reply:
🙏 please remove it

Reply:
Thanks for the suggestion, it's removed